functions/index.js

```diff
@@ -45,59 +58,65 @@ exports.saveAudioSuggestions = functions.https.onRequest(async (req, res) => {
 
 exports.initSettings = functions.https.onRequest(async (req, res) => {
   return cors(req, res, async () => {
-    console.log("Initializing settings");
-    try {
-      const querySnapshot = await admin.firestore().collection("app_settings").doc("default").create({
-        privacy_policy: "",
-        logo_image_id: "",
-        app_enabled: true,
-        app_name: "",
-        app_url: "",
-        translation_language: "",
-        primary_language: "",
-      });
-      res.status(200).send(JSON.stringify("Settings initialized."));
-      console.log("Settings initialized.");
-    } catch (err) {
-      console.log("Error initializing settings:", err);
-      res.status(404).send("Error initializing settings");
-    }
+      const docRef = admin.firestore().collection(SETTINGS.COLLECTION_NAME).doc(SETTINGS.DOCUMENT_NAME);
```

Comment:
Sorry, already merged, but do you mind adding the permissions check that's on the other cloud functions to these new ones?

Something like: 

const hasAccess = await checkAccess_(req, res);
    if (!hasAccess) {
      return;
    }